internal/engine/upper.go

```diff
@@ -286,6 +287,19 @@ func handleBuildCompleted(ctx context.Context, engineState *store.EngineState, c
 		}
 	}
 
+	if mt.Manifest.IsDC() {
+		state, _ := ms.ResourceState.(dockercompose.State)
+
+		cid := cb.Result.AsOneResult().ContainerID
+		spew.Dump(cb.Result)
+		if cid != "" {
+			state = state.WithContainerID(cid)
+			state = state.WithStatus(dockercompose.StatusUp)
```

Comment:
🤔 but how do we know that the container is up?

Reply:
good comment! but whether or not we have a cid doesn't actually tell us whether it's the first build, so the logic is misleading. what do you think of:
> If we have a container ID and no status yet, set status to Up (this is an expected case when we run `docker-compose up` and the service is already running, and we won't get an event to tell us so). If the container is crashing we will get an event subsequently.

Reply:
should also attach the containerId to the state.